Source/Extensions/Blazorise.Video/Video.razor.cs

```diff
@@ -280,6 +280,13 @@ public async Task UpdateSource( VideoSource source, VideoProtectionType protecti
     /// <returns>A task that represents the asynchronous operation.</returns>
     public Task ClearTextTracks() => JSModule.ClearTextTracks( ElementRef, ElementId ).AsTask();
 
+    /// <summary>
+    /// Dispatch an event to change the media playback rate.
+    /// </summary>
+    /// <param name="playbackRate">A double representing the new playback rate. A value of 1.0 represents normal speed, 0.5 is half speed, and 2.0 is double speed.</param>
+    /// <returns>A task that represents the asynchronous operation.</returns>
+    public Task SetPlaybackRate( double playbackRate ) => JSModule.SetPlaybackRate( ElementRef, ElementId, playbackRate ).AsTask();
```

Comment:
Any reason why it can't be kept as `ValueTask`?

Reply:
We usually expose `Task` only. That's it.